pyvcloud/schema/vcd/v1_5/schemas/vcloud/networkType.py

```diff
@@ -13860,12 +13860,12 @@ def buildChildren(self, child_, node, nodeName_, fromsubclass_=False):
     'ParentNetwork': ReferenceType,
     'References': ReferencesType,
     'SubAllocations': SubAllocationsType,
-    'AllocatedIpAddresses': IpAddressesType,
+    'AllocatedIpAddresses': AllocatedIpAddressType,
```

Comment:
In the original XSD, AllocatedIPAddresses is defined twice, so I think it picked up the last definition (IpAddressesType). If it needs to be changed, it should be to 'AllocatedIpAddressesType, not to AllocatedIpAdressType.
Once this is changed, we need to test if other parts of the code expect the type to be IpAddressesType, as originally auto-generated